routes/v1.js

```diff
@@ -182,7 +195,7 @@ api.declare({
       throw err;
     }
     return res.status(409).json({
```

Comment:
Not good enough, we have to load the existing one and check that if it matches... if it matches, then we assume previous request failed, and continue as if we were there... Idempotency...